python_modules/dagster/dagster/_core/executor/step_delegating/step_delegating_executor.py

```diff
@@ -311,6 +311,9 @@ def execute(self, plan_context: PlanOrchestrationContext, execution_plan: Execut
                     else:
                         max_steps_to_run = None  # disables limit
 
+                    # process events from concurrency blocked steps
+                    list(active_execution.concurrency_event_iterator(plan_context))
```

Comment:
looks like were in a generator so not sure why the existing pattern is to consume in place instead of yield, but i think its fine to continue the pattern